ush/python/pygfs/task/land_analysis.py

```diff
@@ -187,6 +187,7 @@ def prepare_IMS(self) -> None:
         os.symlink(exe_src, exe_dest)
 
         # execute CALCFIMSEXE to calculate IMS snowdepth
+        os.chdir(localconf.DATA)
```

Comment:
Shouldn't you already be in `DATA`?

Reply:
This is for the enkfgdas run, and the DATA are different for the different ensemble members.

Reply:
We will have `RUN=enkfgdas` for the ensemble component.

Reply:
@aerorahul Your this comment is for the above conversation, correct? If `RUN=enkfgdas` is available, I can change `CDUMP` back to `RUN`.

Reply:
Removed `os.chdir(localconf.DATA)` for now.